src/app/report/report.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="report-tab row h-100">
   <div class="resizable-horizontal w-25 p-2 border-right h-100">
-    <jqxTree #treeReference (onItemClick)="selectReport($event)" [auto-create]="false"></jqxTree>
+    <jqxTree #treeReference (onItemClick)="selectReport($event)" [auto-create]="false" data-cy-debug="jqxTree"></jqxTree>
```

Comment:
Seeing as this tree is in the report component and not the debug, it should be:
```suggestion
    <jqxTree #treeReference (onItemClick)="selectReport($event)" [auto-create]="false" data-cy-report="jqxTree"></jqxTree>
```

Reply:
added the change